### PR TITLE
#37967: Add small introduction

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: using-valtimo/document/custom-case-headers.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Custom case headers
 
+This page show how a header, that is shown above every case, can be customized.
```
Comment (theo-ritense) [inline using-valtimo/document/custom-case-headers.md:3]:
```suggestion
This page explains how a header that is shown above every case can be customised.
```

File: using-valtimo/document/custom-case-list-columns.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Custom case list columns
 
+This page show how the columns, that are shown in the list of cases, can be changed.
```
Comment (theo-ritense) [inline using-valtimo/document/custom-case-list-columns.md:3]:
```suggestion
This page explains how the columns that are shown in the list of cases can be changed.
```

File: using-valtimo/document/custom-case-headers.md (hunk 1 above)
Comment (nynke-b) [reply using-valtimo/document/custom-case-headers.md]: @theo-ritense I agree with the edit except customised should be customized. According to our style guide we use American spelling: https://docs.valtimo.nl/readme/style-guide

File: using-valtimo/document/custom-case-headers.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Custom case headers
 
+This page explains how a header that is shown above every case can be customised.
```
Comment (Klaas-Ritense) [inline using-valtimo/document/custom-case-headers.md:3]:
```suggestion
This page explains how a header that is shown above every case can be customized.
```

**Review by theo-ritense — APPROVED** [review]

**Review by nynke-b — APPROVED** [review]

**Pull request merged**